config/jobs/kubernetes-sigs/structured-merge-diff/structured-merge-diff-ci.yaml

```diff
@@ -0,0 +1,19 @@
+periodics:
+- name: ci-structured-merge-diff-test
+  interval: 1h
```

Comment:
QQ: do we expect this to change often enough & have flaky enough tests to warrant testing every hour? Seems like we should reduce this or switch to a postsubmit

Reply:
placed a `/hold` if you think this is reasonable, go for `/hold cancel`. I suspect it's a bit high, but it shouldn't be a problem :-)